src/getAuc.ts

```diff
@@ -0,0 +1,24 @@
+import mean from 'ml-array-mean';
+
+import { Curve } from '../types/Curve';
+
+/**
+ * Returns the Area under the curve.
+ * @param curve Object containing the true positivie and false positive rate vectors.
```

Comment:
Variable should be ' curves' and not 'curve'. Please also have a look at other jsdoc n case.